andela/ah-frontend-poseidon
# #162162980 List bookmarked articles

**What does this PR do?**
- Add `bookmarks` button to the profile dropdown
- Implement listing of bookmarked articles
- Fix bug for adding an image when creating an article
- Write tests

**Description of the tasks to be completed?**
- Create `bookmarksActions` file
- Update `ArticleReducer` in the reducers file.
- Create `SidePanel` component
- Add styles for `SidePanel` in `Dashboard.scss`.

**How to manually test it**
- Clone the repository by running `git clone https://github.com/andela/ah-frontend-poseidon.git`
- Run `git checkout ft-list-Bookmarked-articles-162162980`
- Start the server by running `yarn start`
- Sign in by navigating to `/signup` and click on `login`
- On the dashboard, click the profile dropdown and click `Bookmarks`
- You should see a side panel appear with links to your bookmarks.


**Screenshots**
<img width="1183" alt="screen shot 2019-02-01 at 10 12 54 am" src="https://user-images.githubusercontent.com/36928255/52108145-0bbd6200-260a-11e9-8056-90ed62cd368f.png">




**What are the relevant pivotal tracker stories?**
 #162162980
